CSV/boxes.csv

```diff
@@ -82,11 +83,14 @@ iprp,Item Properties Box,ISO
 iref,Item reference,ISO
 j2kH,JPEG 2000 header info,J2KHEIF
 j2kP,JPEG 2000 prefix,J2KHEIF
+jbrd,JPEG bitstream reconstruction data,JPEG XL
 jP$20$20,JPEG 2000 Signature,JPEG2000
 jp2c,JPEG 2000 contiguous codestream,JPEG2000
 jp2h,Header,JPEG2000
 jp2i,intellectual property information,JPEG2000
+jumb,JPEG Universal Metadata Box Format (JUMBF) as described in ISO/IEC 19566-5,JPEG XL
```

Comment:
Is the box defined in JPEG-XL or 19566-5?

Reply:
The contents of the box is defined in JPEG Systems (19566-5), while JPEG XL (18181-2) defines the file format that may contain this box. I'm not sure which of these would be the one to put in the last column.

Reply:
(without reading either) we normally link to the spec. that has the formal definition; may specs use, only one should define

Reply:
Please update the PR to only refer to JPEG Systems